ui/src/app/workflows/components/workflows-list/workflows-list.tsx

```diff
@@ -330,7 +343,23 @@ export function WorkflowsList({match, location, history}: RouteComponentProps<an
                     )}
                 </div>
             </div>
-            <SlidingPanel isShown={!!getSidePanel()} onClose={() => navigation.goto('.', {sidePanel: null})}>
+            <SlidingPanel
+                isShown={!!getSidePanel()}
+                onClose={() => {
+                    // Remove any lingering query params
+                    const qParams: {[key: string]: string | null} = {
```

Comment:
This is duplicate can be removed.

Reply:
Could you explain why this is necessary? The `navigation.goto()` function already has logic to automatically strip out the query parameters if there's been a path change: https://github.com/argoproj/argo-ui/blob/54f36c723d9a0e5d3386689298cbb9c27767fa00/src/components/navigation.ts#L20-L21

If there hasn't been a path change, I think it's fine to leave these query parameters in.

Reply:
So the workflow this would fix is
 - Open the submit page prefilled through the URL
 - Close the submit page
 - Click on Submit again, 
 
 In this case, if the parameters are preserved, we would open the prefilled page. Removing the parameters will ensure the developer can start the page fresh without pre filled data.

Reply:
Okay, thanks. IMO, it'd be better to remove this, because if a developer wants to start fresh, they can click on the "Workflows" icon on the left bar (or navigate to another page and back again)

Reply:
Okay, that makes sense. I checked and this is the only page using a`<SlidingPanel>` that calls `navigation.goto()` on close. Every other page just sets a parameter on close, e.g. https://github.com/argoproj/argo-workflows/blob/e09d2f1121c142bc47cc12f543b1790a92baf8d1/ui/src/app/cluster-workflow-templates/cluster-workflow-template-details.tsx#L145

I'm not sure why this page is special in that respect, but it could be a developer experience thing